languages/php/menhir/parser_php.mly

```diff
@@ -904,7 +904,9 @@ return_type: ":" type_php                 { $1, $2 }
 (* Attributes *)
 (*************************************************************************)
 (* PHP 8 extension (was using << >> in HPHP) *)
-attributes: "#[" listc(attribute) "]" { ($1, $2, $3) }
+attributes:
+| "#[" listc(attribute) "]" { ($1, $2, $3) }
+| attributes attributes {match ($1, $2) with ((op,xs1,cl),(_,xs2,_)) -> (op,xs1@xs2,cl)}
```

Comment:
we usually use lp/rp (left paren, rparen), and in that case
we probably want the left paren from $1 and right paren of $2, so that we get
the right range.